CODEOWNERS

```diff
@@ -27,6 +27,10 @@
 /src/flb_signv4.c        @pettitwesley
 /src/flb_lib.c           @edsiper @niedbalski
 
+@ Core: AWS Auth & Utils
+# ------------
+/src/aws
```

Comment:
I think you are missing @PettitWesley here

Reply:
oops. fixed.